pkgs/applications/science/electronics/qucs-s/default.nix

```diff
@@ -1,27 +1,43 @@
-{ stdenv, lib, fetchFromGitHub, flex, bison, qt4, libX11, cmake, gperf, adms,
-ngspice, wrapGAppsHook,
-kernels ? [ ngspice ] }:
+{ stdenv
+, lib
+, fetchFromGitHub
+, flex
+, bison
+, qtbase
+, qttools
+, qtsvg
+, qtwayland
+, wrapQtAppsHook
+, libX11
+, cmake
+, gperf
+, adms
+, ngspice
+, kernels ? [ ngspice ]
+}:
 
 stdenv.mkDerivation rec {
   pname = "qucs-s";
-  version = "0.0.22";
+  version = "1.0.1";
 
   src = fetchFromGitHub {
     owner = "ra3xdh";
     repo = "qucs_s";
     rev = version;
-    sha256 = "0rrq2ddridc09m6fixdmbngn42xmv8cmdf6r8zzn2s98fqib5qd6";
+    sha256 = "sha256-IdIq/ZY8y/OJ2E51DwhJvcgT+cMAMLozgSuxn8As6ZY=";
   };
 
-  nativeBuildInputs = [ wrapGAppsHook cmake ];
-  buildInputs = [ flex bison qt4 libX11 gperf adms ] ++ kernels;
+  nativeBuildInputs = [ wrapQtAppsHook cmake ];
+  buildInputs = [ flex bison qtbase qttools qtsvg qtwayland libX11 gperf adms ] ++ kernels;
```

Comment:
flex and bison go to nativeBuildInputs most likely

Reply:
this muts be moved to the correct input

Reply:
Thank you for the hint! I've changed it [the next commit](https://github.com/NixOS/nixpkgs/pull/223378/commits/1ad5d99ecf15e6825579a904f7c92a0ddc39d174) while bumping to 1.0.2.